src/data/markdown/translated-guides/en/06 Testing Guides/04 Running large tests.md

```diff
@@ -123,88 +126,66 @@ Here's a screenshot of 3 terminal sessions showing k6, iftop and htop.
 
-### --compatibility-mode=base
-
-The most impactful option to improve k6 performance is to use [`--compatibility-mode=base`](/using-k6/k6-options/reference#compatibility-mode) to disable the internal [Babel](https://babeljs.io/) transpilation and run a k6 script written in ES5.1.  
-
-
-```bash
-# compatibility-mode=base disables the Babel transpilation and the inclusion of corejs 
-k6 run --compatibility-mode=base yourscript.es5.js
-```
-
-> **Background**
-> 
-> Most k6 script examples and documentation are written in ES6+.
-> 
-> By default, k6 transpiles ES6+ code to ES5.1 using babel and loads corejs to enable commonly used APIs.
-> This works very well for 99% of use cases, but it adds significant overheard with large tests.
-> 
-> When running a ES5.1 script instead of the original ES6+ script, k6 can use about 50-85% of memory and significantly reduce the CPU load and startup time.
-
-You can use [webpack](https://webpack.js.org/) to transpile the scripts outside of k6. We have prepared a [webpack.config example](https://github.com/grafana/k6-hardware-benchmark/blob/master/webpack.config.js) that transforms ES6+ code to ES5.1 code for k6. 
-
-In the [k6-hardware-benchmark](https://github.com/grafana/k6-hardware-benchmark) repository, you can use it as follows: 
-
-```bash
-git clone https://github.com/grafana/k6-hardware-benchmark/
-cd k6-hardware-benchmark
-yarn install
-
-yarn run to-es5 someplace/yourscript.js
-# builds the ES5 script in someplace/yourscript.es5.js
-
-k6 run --compatibility-mode=base someplace/yourscript.es5.js
-```
-
 ### discardResponseBodies
 
-You can tell k6 to not process the body of the response by setting `discardResponseBodies` in the options object like this:
+k6 by default loads the response body of the request into memory. This causes much higher memory consumption and is often not necessary.
+You can tell k6 to not process the body of all responses by setting `discardResponseBodies` in the options object like this:
 
 ```javascript
 export const options = {
   discardResponseBodies: true,
 };
 ```
 
-k6 by default loads the response body of the request into memory. This causes much higher memory consumption and often is completely unnecessary.
-If you need response body for some requests you can set [Params.responseType](/javascript-api/k6-http/params).
+If you need the response body for some requests you can override this with [Params.responseType](/javascript-api/k6-http/params).
+
 
 ### --no-thresholds --no-summary
 
-If you are running a local test and streaming results to the cloud (`k6 run -o cloud`), you may want to disable the terminal summary
-and local threshold calculation because thresholds and summary will be displayed in the cloud.
+If you are running a local test and streaming results to the k6 Cloud (`k6 run -o cloud`), you may want to disable the terminal summary
+and local threshold calculation because thresholds and summary will be displayed in the Cloud.
```

Comment:
```suggestion
If you are running a local test and streaming results to the k6 Cloud (`k6 run -o cloud`), you may want to disable the terminal summary
and local threshold calculation,
because k6 Cloud will display the summary and calculate the thresholds.

Without these options, the operations will be duplicated by both the local machine and the k6 Cloud servers.
```